src/config_magic.c

```diff
@@ -829,6 +831,21 @@ TbBool parse_magic_shot_blocks(char *buf, long len, const char *config_textname,
                 break;
             }
           }
+	  case 9: // PUSHONHIT
+		  if (get_conf_parameter_single(buf, &pos, len, word_buf, sizeof(word_buf)) > 0)
+		  {
+			  k = atoi(word_buf);
+			  shotst->old->push_on_hit = k;
+			  n++;
+			  CONFWRNLOG("hnus read \"%s\" parameter in [%s] block of %s file. --->%d",
```

Comment:
Why this warning message? It seems to always give a warning when you have the push-on-hit property, is this just for testing purposes?

Reply:
oh, ye, sry, I forgot to delete it, was in rush.

Reply:
Take your time, we're not on a deadline.